jfrog_platform_self-hosted/manifest.json

```diff
@@ -0,0 +1,81 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "b2748652-b976-461c-91dd-5abd4467f361",
+  "app_id": "jfrog-platform",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "View and analyze JFrog Artifactory and Xray Logs, Violations and Metrics",
+    "title": "JFrog Platform (Self-hosted)",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "JFrog Artifactory Logs dashboard",
+        "image_url": "images/jfrog_artifactory_logs.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "JFrog Logo",
+        "image_url": "images/JFrog+Logo.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "JFrog Artifactory Metrics dashboard",
+        "image_url": "images/jfrog_artifactory_metrics.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "JFrog Xray Logs dashboard",
+        "image_url": "images/jfrog_xray_logs.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "JFrog Xray Violations dashboard",
+        "image_url": "images/jfrog_xray_violations.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "JFrog Xray Metrics dashboard",
+        "image_url": "images/jfrog_xray_metrics.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Containers",
+      "Category::Log Collection",
+      "Category::Metrics",
+      "Category::Security",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
+    ]
+  },
+  "author": {
+    "support_email": "support@jfrog.com",
+    "homepage": "https://github.com/jfrog/log-analytics-datadog",
+    "sales_email": "partners@jfrog.com",
+    "name": "JFrog"
+  },
+  "oauth": {},
+  "assets": {
+    "integration": {
+      "source_type_name": "JFrog Platform",
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "jfrog.",
+        "metadata_path": "metadata.csv"
+      }
+    },
+    "dashboards": {
+      "JFrog Artifactory Logs (Self-hosted)": "assets/dashboards/jfrog_artifactory_logs_(self-hosted).json",
+      "JFrog Artifactory Metrics (Self-hosted)": "assets/dashboards/jfrog_artifactory_metrics_(self-hosted).json",
+      "JFrog Xray Logs (Self-hosted)": "assets/dashboards/jfrog_xray_logs_(self-hosted).json",
+      "JFrog Xray Metrics (Self-hosted)": "assets/dashboards/jfrog_xray_metrics_(self-hosted).json",
+      "JFrog Xray Violations (Self-hosted)": "assets/dashboards/jfrog_xray_violations_(self-hosted).json"
```

Comment:
In order to provide a smooth experience for users, can you continue to use the same `key` as the old manifest file? 
Ex: `Jfrog Artifactory Dashboard`. 

Unfortunately we cannot remove existing dashboards or change the `key` at this time. However the changes to the dashboard.json payloads themselves are still OK!